storage/connectors/jdbc/src/test/java/feast/storage/connectors/jdbc/retriever/JdbcHistoricalRetrieverTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.storage.connectors.jdbc.retriever;
+
+import com.google.protobuf.Duration;
+import feast.proto.core.FeatureSetProto;
+import feast.proto.serving.ServingAPIProto;
+import feast.storage.api.retriever.FeatureSetRequest;
+import feast.storage.api.retriever.HistoricalRetriever;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class JdbcHistoricalRetrieverTest {
+  @Rule public transient TestPipeline p = TestPipeline.create();
+  private HistoricalRetriever postgresqlFeatureRetriever;
+
+  //    Give postgresql writing permission to your staging_location
+  //    run: sudo chmod 777 <staging_location>
+  private String staging_location = System.getenv("staging_location");
+  private String url = "jdbc:postgresql://localhost:5432/postgres";
+  private String class_name = "org.postgresql.Driver";
+  private String username = "postgres";
+  private String pw = System.getenv("postgres_pw");
+  private Map<String, String> postgressqlConfig = new HashMap<>();
+
+  @Test
+  public void dummyTest() {
```

Comment:
i don't think we need this set of code .

Reply:
oh, yes. we dont need dummy tests. let me rm them.

Reply:
fixed. Please review again~